Add principle poster box

HendrikSchmidt opened the pull request.

bastipnt commented on app/components/InfoBox.tsx:
```diff
@@ -42,7 +55,7 @@ const InfoBox = ({
   linkList,
   buttons,
-  Icon,
+  visual,
```
nice, I think I like this naming

bastipnt commented on app/components/InfoBox.tsx:
```diff
@@ -28,11 +26,26 @@ export type InfoBoxProps = {
+};
+
+const imageSizes = {
+  icon: "size-80",
```
makes sense to call it icon and not xs 👍

bastipnt commented on app/components/PrinciplePosterBox.tsx:
```diff
@@ -0,0 +1,41 @@
+            {
+              title: principlePosterBox.downloadTitle,
+              url: principlePosterBox.downloadUrl,
+              download: true,
```
Linter is complaining, that `download` does not exist in the props 🤷

bastipnt commented on .talismanrc:
```diff
@@ -4,4 +4,6 @@ fileignoreconfig:
 - filename: app/root.tsx
-  allowed_patterns: [posthogKey]
\ No newline at end of file
+  allowed_patterns: [posthogKey]
```
does this belong to this pr?

HendrikSchmidt replied:
I think this is only highlighted as added because I put a newline at the end :)

bastipnt reviewed (changes requested):
It already looks really good, just two things:
1. The `Image` should move to the bottom of the `InfoBox` on mobile, according to figma: https://www.figma.com/design/HiiKWvM5xBlNCM9wXYvInb/%F0%9F%9F%A6-Digitalcheck-Komponenten-Library?node-id=1334-7921
2. The `Image` on `InfoBox` is not fully responsive: <img width="1348" height="2398" alt="Image" src="https://github.com/user-attachments/assets/2677767e-e578-47be-9219-1d6019c7a8f0" />
Apart from that nice work :)

HendrikSchmidt commented:
> It already looks really good, just two things:
> 
> 1. The `Image` should move to the bottom of the `InfoBox` on mobile, according to figma: https://www.figma.com/design/HiiKWvM5xBlNCM9wXYvInb/%F0%9F%9F%A6-Digitalcheck-Komponenten-Library?node-id=1334-7921
> 2. The `Image` on `InfoBox` is not fully responsive

1. Changed the `flex-direction` to achieve this.
2. Added responsive image sizing for large images.
